packages/keystone/lib/Keystone/index.js

```diff
@@ -111,7 +111,9 @@ module.exports = class Keystone {
         })
       )
     ).then(() => {
-      if (this.eventHandlers.onConnect) this.eventHandlers.onConnect(this);
+      if (this.eventHandlers.onConnect) {
+        return this.eventHandlers.onConnect.call(this, this);
```

Comment:
Making sure to return any promises so the `.connect()` call will wait.

Reply:
I agree with returning it for that reason but why call it and set the `this` value? I feel like that just adds confusion because then there's another way to get the keystone instance.

Reply:
Whoops, that was me trying something and I forgot to take it out. Thanks for the catch!